staging/src/k8s.io/kubectl/pkg/cmd/version/version.go

```diff
@@ -101,7 +105,10 @@ func (o *Options) Complete(f cmdutil.Factory, cmd *cobra.Command) error {
 }
 
 // Validate validates the provided options
-func (o *Options) Validate() error {
+func (o *Options) Validate(args []string) error {
+	if len(args) != 0 {
+		return errors.New(`extra arguments`)
```

Comment:
It would be better to use double quote because we don't have anything to escape in that string.

Reply:
I think it would be better to return `cmdutil.UsageErr` like in here https://github.com/kubernetes/kubernetes/blob/9b09612d1b0a6ec441833f928cbb205d936da59c/staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go#L335

Reply:
Fixed